cli/internal/cmd/configgenerate.go

```diff
@@ -51,6 +51,22 @@ func configGenerate(cmd *cobra.Command, fileHandler file.Handler, provider cloud
 		return err
 	}
 
+	_, err = createConfig(cmd, fileHandler, provider, flags.file)
+	if err != nil {
+		return err
+	}
+
+	if flags.file != "-" {
+		cmd.Println("Config file written to", flags.file)
+		cmd.Println("Please fill in your CSP-specific configuration before proceeding.")
+		cmd.Println("For more information refer to the documentation:")
+		cmd.Println("\thttps://docs.edgeless.systems/constellation/getting-started/first-steps")
+	}
+	return nil
+}
+
+// createConfig creates a config file for the given provider and writes it to the given outputFile. If the outputFile is "-", the config is written to stdout.
+func createConfig(cmd *cobra.Command, fileHandler file.Handler, provider cloudprovider.Provider, outputFile string) (*config.Config, error) {
```

Comment:
Imo it would be a bit cleaner to just generate the config in `createConfig` and do the output in `configGenerate`. This would also make the overwrite in iamcreate unnecessary.